css/css-contain/container-queries/var-evaluation.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<title>CSS Container Queries Test: var()</title>
+<link rel="help" href="https://drafts.csswg.org/css-contain-3/#size-container">
```

Comment:
Needs to be css-conditional-5 now. Also this file needs to move into the css-conditional/container-queries directory.